dnp3-master/service/start_service.py

```diff
@@ -119,17 +159,98 @@ def run_master(device_ip_port_config_all_13bus, names,simulation_id,gapps,dnp3_t
 
     msg_count=0
     csv_dict = {}
-    cim_full_msg = {'simulation_id': simulation_id, 'message':{'timestamp': int(time.time()),'measurements':{}}}
+    cim_full_msg = {'message':{'timestamp': int(time.time()),'measurements':{}}}
+    
 
+
+
+
+    # while True:
+    #     current_time = time.time()
+    #     for master in masters:
+    #         master.send_scan_all_request()
+    #         # master_soe_handler: SOEHandler = master.soe_handler
+    #         # cim_msg = master_soe_handler.get_msg()
+    #         cim_msg = master.soe_handler.db  # TODO: note this is place where cim_msg should be formatted
+    #         # cim_msg = master.master_application.get_config()
+    #         cim_full_msg['message']['measurements'].update(cim_msg)
+    #         cim_full_msg['message']['timestamp'] = str(int(current_time))
+    #         _log.debug(f'Publishing CIM measurement: {json.dumps(cim_full_msg)}')
+    #         # gapps.send('/topic/goss.gridappsd.field.output', json.dumps(cim_full_msg))
+    #         gapps.send(topics.field_output_topic(), json.dumps(cim_full_msg))
+    #     logger.info(f"{cim_full_msg=}")  # TODO: kefei added
+    #     msg_count+=1
+    #     time.sleep(2)
+    
+    # """ # example message: https://gridappsd.readthedocs.io/en/master/using_gridappsd/index.html?highlight=cim#subscribe-to-simulation-output
+    {
+        "simulation_id" : "12ae2345",
+        "message" : {
+            "timestamp" : "1357048800",
+            "measurements" : {
+                "123a456b-789c-012d-345e-678f901a234b":{
+                                    "measurement_mrid" : "123a456b-789c-012d-345e-678f901a234b",
+                                    "value": 1
+                                    # "magnitude" : 3410.456,
+                                    # "angle" : -123.456
+                                    }
+                            }
+                    }
+    }
+
+
+    # in /home/shared_user/gridappsd-dnp3-master/dnp3-master/service/config/new_measurement_dict_master.json
+    new_measurement_dict_master = {
+        "ufls_59.1": {
```

Comment:
It would be better if python files do not have a specific model dependent dictionaries. Make that configurable based on input.